#! /usr/bin/env python

PACKAGE="eus_imitation"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

#        name        type     level    description       default  min  max
gen.add("clip_threshold", double_t, 0, "Threshold of Clip Inference result.", 0.8, 0.0, 1.0)
gen.add("hand_dist_threshold", double_t, 0, "Threshold of Hand Distance.", 0.09, 0.0, 0.2)
gen.add("objects",   str_t, 0, "Objects in interaction. You can split objects by ;", "cup; bottle;")
exit(gen.generate(PACKAGE, "eus_imitation", "HandDetection"))
